src/pages/Landing/index.tsx

```diff
@@ -56,6 +56,29 @@ export default () => {
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
 
   if (isLoading) return <Loader />;
+  function handleDelete(customerId: any): void {
+    const deleteCustomer = () => {
+      const customers = customersGetter();
+      const index = customers.findIndex((customer) => customer.name === customerId);
+
+
+      if (index !== -1) {
+        const updatedCustomers = [...customers];
+        updatedCustomers.splice(index, 1);
+        customersSetter(updatedCustomers);
+      }
+    };
```

Comment:
(deleted my previous comment because it was referencing too much code.  

This should really exist in src/api/CustomerApi.ts    You already have a similar method there, so you can just use that and adjust it as needed.

Reply:
reminder about this comment.  You are very close!

Reply:
2nd reminder about this comment, you should be able to call deleteCustomer() from the CustomerApi.tsx file